src/site/layouts/landing_page.drupal.liquid

```diff
@@ -6,7 +6,18 @@
   <main>
     <div class="usa-grid usa-grid-full">
       <article class="usa-width-two-thirds">
+        {% if fieldTitleIcon %}
+          <div class="inline hub-main-icon">
+            <i class="icon-large white hub-icon-{{ fieldTitleIcon }} hub-background-{{ fieldTitleIcon }}"></i>
+          </div>
+
+          <div class="inline hub-main-title">
+            <h1>{{ title }}</h1>
+          </div>
+
+        {% else %}
         <h1>{{ title }}</h1>
+        {% endif %}
```

Comment:
Are there instances where a landing page doesn't have `fieldTitleIcon`? Or could this only happen while we're in dev?

Reply:
This would just be for dev. Most of these landing pages don't have this field populated yet, so included this if statement so the page wouldn't look busted in the meantime.